passes/techmap/abc.cc

```diff
@@ -876,6 +908,13 @@ void abc_module(RTLIL::Design *design, RTLIL::Module *current_module, std::strin
 	fprintf(f, "%s\n", abc_script.c_str());
 	fclose(f);
 
+	buffer = stringf("%s/user.script", tempdir_name.c_str());
```

Comment:
Do we need the `abc.script`/`user.script` separation? Why not write both to the same file?

Reply:
The `abc.script` and `user.script` separation is how it currently works. It basically separates the setup (loading the network and library and later writing it out) from the actual optimization and mapping script.

A different approach (and the one I would like to move in) is to give an option to not use the `abc.script` part and instead expect it to be part of the custom `user.script` and then just replace the placeholders in `user.script`.

This would give full freedom in what people do with ABC, going as far as invoking multiple ABC scripts in parallel and selecting one based one some criteria before returning to Yosys.